modules/templates/helper.go

```diff
@@ -565,6 +565,12 @@ func Avatar(item interface{}, others ...interface{}) template.HTML {
 			return AvatarHTML(src, size, class, user.DisplayName())
 		}
 	}
+	if user, ok := item.(*models.Organization); ok {
```

Comment:
I don't think it should be called "user", let's call it "org".
```suggestion
	if org, ok := item.(*models.Organization); ok {
```